### DISCUSSION TITLE
USB Bandwidth

### DISCUSSION THREAD

**Discussion by dmorilha** [Motion-Project/motion#1745] [General]
First of all, thank you for such an incredible project. I've been using motion since 2012 and it just works great.
Lately I was struggling to get more than one usb camera on a raspberry pi due to the bandwidth limitation pointed out in the documentation.
The solution was to use `mjpeg-streamer` (another excellent project) which doesn't suffer from the problem and open the two cameras as netcams in motion.
But I just realized the same can be achieved by setting `v4l2` `mjpeg` `palette` in `video_params`. So perhaps the documentation could be updated with that trick.
Thank you all again!

**Comment by tosiara** [comment]
But it is already mentioned on the page of uvcvideo: https://www.ideasonboard.org/uvc/faq/

**Comment by dmorilha** [comment]
You're right

**Comment by tytrhex** [comment]
> First of all, thank you for such an incredible project. I've been using motion since 2012 and it just works great.
> Lately I was struggling to get more than one usb camera on a raspberry pi due to the bandwidth limitation pointed out in the documentation.
> The solution was to use `mjpeg-streamer` (another excellent project) which doesn't suffer from the problem and open the two cameras as netcams in motion.
> But I just realized the same can be achieved by setting `v4l2` `mjpeg` `palette` in `video_params`. So perhaps the documentation could be updated with that trick.
> Thank you all again!


Dude! You totally saved me a lot a time. Just setted v4l2 palette and its working. Before I used orange pi PC to just stream the cam to the pc. Thank you again sir!

**Comment by dmorilha** [comment]
not a problem and glad I could help, the issue is not trivial to figure out, and it should be made more explicit in the logs.
